### DISCUSSION TITLE
thelogrus Python package - required or purpose?

### DISCUSSION THREAD

**Discussion by darloxflyer** [unixorn/ha-mqtt-discoverable#254] [Q&A] [answered]
Thanks for a nifty helper library here!  Only one question -- the dependencies specify installation of "thelogrus", which seems to be a random collection of unrelated functions, some of which look a bit questionable from a device security standpoint.  Also, in searching through the ha-mqtt-discoverable code, it doesn't look like that library is actually USED anywhere.

Is that just a relic/erroneous dependency, or is it required somewhere I'm just not seeing?  Thanks in advance!

**Comment by unixorn** [comment, marked as answer]
It's a relic from when the CLI programs were in the same repository as the library. I'll pull it.
